packages/core/useUrlSearchParams/index.ts

```diff
@@ -44,7 +44,7 @@ export function useUrlSearchParams<T extends Record<string, any> = UrlParams>(
   if (!window)
     return reactive(initialValue) as T
 
-  const state: Record<string, any> = reactive(initialValue)
+  const state: Record<string, any> = reactive({})
```

Comment:
It won't trigger the watcher callback by initializing the reactive state with `{}` in vue2